embassy-stm32/src/rcc/wl.rs

```diff
@@ -116,6 +140,7 @@ pub struct Config {
     pub enable_lsi: bool,
     pub enable_rtc_apb: bool,
     pub rtc_mux: RtcClockSource,
+    pub adc_clock_source: AdcClockSource,
```

Comment:
The adc clock source should be configured in the rcc mod and the clock frequency added to the clocks adc struct.